### PR TITLE
V1 release

### PR DESCRIPTION
**Pull request by oudeismetis**
Connected to #99

### CODE REVIEW

File: franklin/core/urls.py [hunk 1]
```diff
@@ -43,3 +43,7 @@
     # Utilities
     url(r'^health/$', health, name='health'),
 ]
+
+urlpatterns = [
+    url(r'^v1/', include(v1_patterns)),
+]
```
Comment (oudeismetis) [inline franklin/core/urls.py]: `v1` ✨

File: franklin/github/tests.py [hunk 2]
```diff
@@ -1,62 +1,248 @@
...
-            name='istrategylabs', github_id=607333)
-        self.site = Site.objects.create(
-            owner=self.owner, name='franklin-dashboard', github_id=45864453)
+def ordered(obj):
+    if isinstance(obj, dict):
+        return sorted((k, ordered(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(ordered(x) for x in obj)
+    else:
+        return obj
```
Comment (oudeismetis) [inline franklin/github/tests.py]: I didn't want to do this, but for some reason the dict returned from the api response is slightly different, so you can't compare them directly. You have to order the objects first. (see 'case in point' below)

File: franklin/github/tests.py [hunk 3]
```diff
@@ -1,62 +1,248 @@
...
+        post_github = Mock(status_code=200)
+        post_github.json.return_value = {'id': 123}
+        mock_post.return_value = post_github
+
+        expected = get_mock_data('api_responses', 'new_site')
+        with patch.dict('os.environ', {'BASE_URL': 'example.com'}):
+            response = self.client.post(url, {'github': 'isl/bar'},
+                                        **self.header)
+
+        self.assertEqual(ordered(expected), ordered(response.data))
```
Comment (oudeismetis) [inline franklin/github/tests.py]: 'case in point'

File: franklin/github/tests.py [hunk 4]
```diff
@@ -1,62 +1,248 @@
...
+        # Creating webhook and deploy key on github repo
+        mock_post.return_value = Mock(status_code=200)
+
+        expected = get_mock_data('api_responses', 'build')
+        # Build object has an auto-generated creation date. mock that.
+        with patch('django.utils.timezone.now') as mock_now:
+            mock_now.return_value = datetime(2016, 5, 4)
+            response = self.client.post(url, **self.header)
+
+        self.assertEqual(ordered(expected), ordered(response.data))
```
Comment (oudeismetis) [inline franklin/github/tests.py]: This method is a good example of a "complex" end-to-end test that mocks multiple calls to multiple external dependencies. Also mocks django's auto generated datetime for a model save.

File: franklin/core/urls.py (hunk 1 above)
Comment (camiriveron) [inline franklin/core/urls.py]: ✨ ✨ ✨

**Pull request merged**